src/libraries/System.Private.CoreLib/src/System/TimeOnly.cs

```diff
@@ -0,0 +1,973 @@
+                result = default;
+                return false;
+            }
+
+            return TryParseExact(s.AsSpan(), format.AsSpan(), provider, style, out result);
+        }
+
+        /// <summary>
+        /// Converts the specified string of a time to its TimeOnly equivalent and returns a value that indicates whether the conversion succeeded.
+        /// </summary>
+        /// <param name="s">The string containing time to parse.</param>
+        /// <param name="formats">An array of allowable formats of s.</param>
+        /// <param name="result">When this method returns, contains the timeOnly value equivalent to the time contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s parameter is Empty, or does not contain a valid string representation of a time. This parameter is passed uninitialized.</param>
+        /// <returns>true if the s parameter was converted successfully; otherwise, false.</returns>
+        public static bool TryParseExact(string s, string [] formats, out TimeOnly result) => TryParseExact(s, formats, DateTimeFormatInfo.CurrentInfo, DateTimeStyles.None, out result);
+
+        /// <summary>
+        /// Converts the specified string of a time to its TimeOnly equivalent and returns a value that indicates whether the conversion succeeded.
+        /// </summary>
+        /// <param name="s">The string containing the time to parse.</param>
+        /// <param name="formats">An array of allowable formats of s.</param>
+        /// <param name="provider">An object that supplies culture-specific formatting information about s.</param>
+        /// <param name="style">A bitwise combination of enumeration values that defines how to interpret the parsed date. A typical value to specify is None.</param>
+        /// <param name="result">When this method returns, contains the TimeOnly value equivalent to the time contained in s, if the conversion succeeded, or MinValue if the conversion failed. The conversion fails if the s parameter is Empty, or does not contain a valid string representation of a time. This parameter is passed uninitialized.</param>
+        /// <returns>true if the s parameter was converted successfully; otherwise, false.</returns>
+        public static bool TryParseExact(string s, string [] formats, IFormatProvider? provider, DateTimeStyles style, out TimeOnly result)
+        {
+            if (s == null)
+            {
+                result = default;
+                return false;
+            }
+
+            return TryParseExact(s.AsSpan(), formats, provider, style, out result);
+        }
+
+
+        // Acceptable formats:
+        //      Sortable date/time pattern      s   Time part only
+        //      RFC1123 pattern                 r/R Time part only like o
+        //      round-trip date/time pattern    o/O Time part only
+        //      Short time pattern              t
+        //      Long time pattern               T
```

Comment:
Why not move this in public comments for `public string ToString(string? format)`?

Reply:
will do.